### PR TITLE
Feature lwdev 4113 template and settings

### PR DESCRIPTION
**Pull request by tarurar**

### CODE REVIEW

File: src/Lykke.Job.TradeDataAggregator.Services/HealthService.cs [hunk 1]
```diff
@@ -73,5 +75,23 @@ public string GetHealthWarningMessage()
 
             return null;
         }
+
+        public IEnumerable<HealthIssue> GetHealthIssues()
+        {
+            var issues = new HealthIssuesCollection
+            {
+                { nameof(LastClientsScanningStartedMoment), LastClientsScanningStartedMoment.ToString() },
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.TradeDataAggregator.Services/HealthService.cs]: There are not health issues. Only warningMessage is health issue

File: src/Lykke.Job.TradeDataAggregator.Services/ShutdownManager.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        private readonly ILog _log;
+
+        public ShutdownManager(ILog log)
+        {
+            _log = log;
+        }
+
+        public async Task StopAsync()
+        {
+            // TODO: Implement your shutdown logic here. Good idea is to log every step
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.TradeDataAggregator.Services/ShutdownManager.cs]: ScanClientsHandler should be stopped here

File: src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            _log = log;
+            _scanClientsHandler = scanClientsHandler;
+        }
+
+        public async Task StartAsync()
+        {
+            await _log.WriteInfoAsync(nameof(StartAsync), "", "Starting scan clients handler");
+
+            _scanClientsHandler.Start();
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs:29]: await is missed. And better to rename the `Start` to the `StartAsync`

File: src/Lykke.Job.TradeDataAggregator/Modules/JobModule.cs [hunk 4]
```diff
@@ -11,19 +11,20 @@
...
 {
     public class JobModule : Module
     {
-        private readonly AppSettings _settings;
+        private readonly IReloadingManager<AppSettings> _settings;
         private readonly ILog _log;
         private readonly ServiceCollection _services;
 
-        public JobModule(AppSettings settings, ILog log)
+        public JobModule(IReloadingManager<AppSettings> settings, ILog log)
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.TradeDataAggregator/Modules/JobModule.cs]: Will be better to pass `DbSettings` wrapped into the `IReloadingManager`, but `AppSettings` as raw, unwrapped object

File: src/Lykke.Job.TradeDataAggregator/Startup.cs [hunk 5]
```diff
@@ -47,95 +43,169 @@ public Startup(IHostingEnvironment env)
...
+                Log = CreateLogWithSlack(services, appSettings);
 
-            builder.RegisterInstance(appSettings.RabbitMq).SingleInstance();
+                builder.RegisterModule(new JobModule(appSettings, Log));
 
-            builder.RegisterType<RabbitMqHandler>()
-                .AsSelf()
-                .As<IStartable>()
-                .SingleInstance();
+                builder.AddTriggers();
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.TradeDataAggregator/Startup.cs]: Is triggers still using?

File: src/Lykke.Job.TradeDataAggregator.Services/ShutdownManager.cs (hunk 2 above)
Comment (tarurar) [reply src/Lykke.Job.TradeDataAggregator.Services/ShutdownManager.cs]: Yep

File: src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs (hunk 3 above)
Comment (tarurar) [reply src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs:29]: TimePeriod doesn't have StartAsync method ...

File: src/Lykke.Job.TradeDataAggregator/Startup.cs (hunk 5 above)
Comment (tarurar) [reply src/Lykke.Job.TradeDataAggregator/Startup.cs]: Removed

File: src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs (hunk 3 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Job.TradeDataAggregator.Services/StartupManager.cs:29]: My bad. Looks like I'm just looked at the wrong place

File: src/Lykke.Job.TradeDataAggregator/Modules/JobModule.cs (hunk 4 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Job.TradeDataAggregator/Modules/JobModule.cs]: Example - https://github.com/LykkeCity/Lykke.Service.CandlesHistory/blob/master/src/Lykke.Service.CandlesHistory/DependencyInjection/ApiModule.cs#L41-L43

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**